Python/optimizer.c

```diff
@@ -172,7 +172,7 @@ _Py_SetOptimizer(PyInterpreterState *interp, _PyOptimizerObject *optimizer)
     }
     else if (cold_exits_initialized == 0) {
         cold_exits_initialized = 1;
-        for (int i = 0; i < UOP_MAX_TRACE_LENGTH; i++) {
+        for (int i = 0; i < UOP_MAX_TRACE_LENGTH/4; i++) {
```

Comment:
Since this is repeated, probably worth just defining `#define COLD_EXIT_COUNT ((UOP_MAX_TRACE_LENGTH) / 4)`